drivers/remote/api/api.go

```diff
@@ -16,6 +16,12 @@ func (r *Response) GetError() string {
 	return r.Err
 }
 
+// GetCapabilityResponse is the response of GetCapability request
+type GetCapabilityResponse struct {
+	Response
+	Scope string
```

Comment:
Scope is the only capability that we are interested in.
Just want to make sure that when we add more capabilities, libnetwork must not fail in the future if the users are using an older version of plugin that might not have implemented the newly added capabilities. We just have to make sure that the `driver.call` function doesn't fail.